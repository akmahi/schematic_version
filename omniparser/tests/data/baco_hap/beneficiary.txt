|100112060|02|H|LTest1|FTest1|B||19850214|M||2181 WESTFIELD||Trenton|MI|48183|USA|7344171491||20121001|20170812|Y||||10002788|MANHEIM DETROIT|1000|MANHEIM DETROIT ACTIVE|HHMR0001||COMM|HMO||1000|AA000317|||1770545766|ATHENS, ANDREW A.|||||||||||||||4265129
|100984456|00|M|LTest2|FTest2|T||19470217|F||15477 NORTHVILLE FORES|APT 41D|PLYMOUTH|MI|48170|USA|2482771759||20210101|20211231|Y|5P74WN8CG91|||10000026|NON-GROUP PROGRAM SENIOR PLUS|1006|NON-GROUP PROGRAM SENIOR PLUS MA|MHMI0007||MA|HMO|IPD|1000|MA000252|||1912991894|VALENTI, MATTEO|||||||||||||||10781216
|100116285|00|M|LTest3|FTest3|L||19481203|F||16239 TAFT ST||ROMULUS|MI|48174|USA|7347715452||20200101|20681231|Y|7TJ3UC2EP87|||10000672|GENERAL MOTORS VEBA HMO|1206|GENERAL MOTORS VEBA PROTECTED SURV SP SEMI MA|MHMG0022||MA|HMO||1000|MA000196|||1760444228|ATHENS, ELLEN K.|||||||||||||||2734540
|100430131|02|W|LTest4|FTest4|||19620411|F||38 CHERRY GROVE||Ecorse|MI|48229|USA|||20120101|20180531|Y||||10000721|SUBURBAN MOBILITY AUTHORITY HMO|1201|SMART B-RETIREE|HHMR0001||COMM|HMO||1000|AA000395|||1659356400|ZAKOLSKI, DOUGLAS J.|||||||||||||||1471347
|101336074|01|W|LTest5|FTest5|||19610929|F||2105 OLD GODDARD ROAD||LINCOLN PARK|MI|48146|USA|3133183138||20210401|20811231|Y||||10002562|DEARBORN HOLDING COMPANY LLC|1100|DEARBORN HOLDING COMPANY|HHMR0001||COMM|HMO||1000|AA000597|||1568413367|JURASEK, HENRY M.|||||||||||||||11321226
|100675472|00|M|LTest6|FTest6|||19890516|F||1708 KINGS HIGHWAY||LINCOLN PARK|MI|48146|USA|||20150601|20170430|Y||||10000011|FORD MOTOR COMPANY HMO|1009|FORD MOTOR COMPANY - SALARIED HE COBRA|HEMR0001||COMM|HMO|EE|1000|AA000901|||1629017207|BLANZY, JOHN E.|||||||||||||||4131249
|100135828|00|M|LTest7|FTest7|||19400820|M||1733 HEATHER DR||TEMPERANCE|MI|481829422|USA|7342247818||20210101|20211231|Y|3KV9XE7NN45|||10000026|NON-GROUP PROGRAM SENIOR PLUS|1006|NON-GROUP PROGRAM SENIOR PLUS MA|MHPI0005||MA|HMO|IPD|1000|MA000258|||1760536528|SEKHON, AVTAR S.|||||||||||||||2992339
|100040659|02|S|LTest8|FTest8|L||20020803|M||24823 COBBLESTONE CT||FARMINGTON HILLS|MI|48336|USA|3138150119||20120101|20171231|Y||||10000703|FCA US LLC HMO|1000|FCA US LLC ACTIVE|HHMR0001||COMM|HMO||1000|AA000722|||1699904086|CREGER, CLARK M.|||||||||||||||3515461
|100263770|03|S|LTest9|FTest9|C||19970612|M||9180 KINLOCH||REDFORD|MI|48239|USA|7348377949||20110101|20171231|Y||||10000374|CADON PLATING HMO|1000|CADON PLATING HMO ACTIVE|HHMR0001||COMM|HMO||1000|AA000146|||1598053142|VINTON, BRIAN A.|||||||||||||||3938027
|100874961|01|S|LTest10|FTest10|||19931105|M||17567 CRANBROOK DR||NORTHVILLE|MI|481684331|USA|2483440077||20170101|20171231|Y||||10000005|HAP HMO INDIVIDUAL|1000|ON EXCHANGE|HYMR0310||COMM|HMO|IX|1000|AA002587|||1942433289|KOTTUKAPALLY, NOBIN J.|||||||||||||||10621509
|100008409|00|M|LTest11|FTest11|J||19900512|F||24794 STREWING RD||BROWNSTOWN|MI|48134|USA|7347759091||20210101|21101231|Y||||10000017|FEDERAL EMPLOYEES HEALTH BENEFITS PROG.|5600|FEHB-97381600 DEPT VET AFFAIRS ACTIVE|HHMR0001||COMM|HMO||1000|AAS00192|||1154329423|OOSTENDORP, MICHAELE C.|||||||||||||||1119392
|100821488|00|M|LTest12|FTest12|||19390509|M||9583 GRANDMONT||DETROIT|MI|48227|USA|||20180101|20181231|Y|5XD5U79JV67|||10000026|NON-GROUP PROGRAM SENIOR PLUS|1006|NON-GROUP PROGRAM SENIOR PLUS MA|MHMI0007||MA|HMO|IPD|1000|MA000205|||1790764587|ARUNACHALAM, VIJAYALAKSHMI|||||||||||||||2618124
|100219147|00|M|LTest13|FTest13|M||19590831|M||3811 BENNET ST||DEARBORN|MI|48124|USA|3132773162||20200401|20791231|Y||||10002581|ROBERT BOSCH CORPORATION HMO|1000|ROBERT BOSCH CORPORATION ACTIVE|HHMR0001||COMM|HMO||1000|AAS00160|||1033189477|MEHDI, FARAH|||||||||||||||3762354
|100144046|00|M|LTest14|FTest14|R|JR|19550121|M||6816 SOUTH STONEY CREEK||MONROE|MI|48162|USA|7348721190||20120101|20171231|Y||||10000703|FCA US LLC HMO|1000|FCA US LLC ACTIVE|HHMR0001||COMM|HMO||1000|AA000722|||1184704181|BELL, STEPHEN R.|||||||||||||||3092206
|100106231|00|M|LTest15|FTest15|L||19560921|M||3045 LAFAYETTE||TRENTON|MI|48183|USA|7343082899||20210901|20761231|Y|8QE1EX8WV98|||10000673|CHRYSLER VEBA HMO|1206|CHRYSLER HOURLY VEBA GENERAL SEMI MA|MHMG0022||MA|HMO||1000|MA000199|||1568413367|JURASEK, HENRY M.|||||||||||||||2559638
|100081887|00|M|LTest16|FTest16|W|JR|19561005|M||26620 LARAMIE||FLAT ROCK|MI|48134|USA|3136082509||20120401|20181231|Y||||10000586|METAL TECH PRODUCTS|1000|METAL TECH PRODUCTS ACTIVE|HHMR0007||COMM|HMO1||1000|AA001220|||1164464301|NELSON, CAROLYN D.|||||||||||||||2207379
|100300283|00|M|LTest17|FTest17|||19821014|M||25616 KINYON ST||TAYLOR|MI|48180|USA|7349341630||20211201|21021231|Y||||10002782|WYANDOTTE ALARM COMPANY|1100|WYANDOTTE ALARM COMPANY HMO 0 ACTIVE|HBMR0035||COMM|HMO|X|1000|AAQ00165|||1609818582|ROSSI, BENJAMIN M.|||||||||||||||4062868
|100110932|00|M|LTest18|FTest18|VENUS|Y||19730105|F||9336 COLUMBIA||REDFORD|MI|48239|USA|3137217050||20130601|20931231|Y||||10000664|WAYNE STATE UNIVERSITY HMO|1100|WAYNE STATE UNIVERSITY -PLAN 1 ACTIVE|HHMR0001||COMM|HMO||1000|AA001196|||1821042581|ELMENINI, MOHAMMAD S.|||||||||||||||2650317
|100930402|00|M|LTest19|FTest19|J||19721124|M||41486 SIMCOE DR||CANTON|MI|48188|USA|||20170701|20180630|Y||||20000923|CALO AND SONS CONSTRUCTION INC|1000|CALO AND SONS CONSTRUCTION HMO 2500 PLUS ACTIVE|HBMR0133||COMM|HMO|X|1000|AA002661|||1780657569|SCHAEFFER, MICHAEL S.|||||||||||||||10699845
|100162768|01|D|LTest20|FTest20|A||20011024|F||527 IDA MAYBEE RD||MONROE|MI|48161|USA|3135873665||20190101|20220611|Y||||10000011|FORD MOTOR COMPANY HMO|1000|FORD MOTOR COMPANY - HOURLY ACTIVE|HHMR0001||COMM|HMO||1000|AAS00109|||1043660640|ELLADKI, RACHA|||||||||||||||3349083
