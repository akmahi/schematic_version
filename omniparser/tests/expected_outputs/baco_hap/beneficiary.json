[
	{
		"patient": {
			"address": {
				"city": "Trenton",
				"country": "USA",
				"line1": "2181 WESTFIELD",
				"state": "MI",
				"zip": "48183"
			},
			"dob": "19850214",
			"first_name": "FTest1",
			"gender": "M",
			"insurance_plan": {
				"group_id": "10002788",
				"group_name": "MANHEIM DETROIT",
				"plan_id": "HHMR0001",
				"subgroup_id": "1000",
				"subgroup_name": "MANHEIM DETROIT ACTIVE"
			},
			"last_name": "LTest1",
			"member_suffix": "02",
			"middle_initial": "B",
			"phone_home": "7344171491",
			"provider": {
				"last_name": "ATHENS, ANDREW A.",
				"npi": "1770545766"
			},
			"relationship_code": "H",
			"subscriber_id": "100112060"
		}
	},
	{
		"patient": {
			"address": {
				"city": "PLYMOUTH",
				"country": "USA",
				"line1": "15477 NORTHVILLE FORES",
				"line2": "APT 41D",
				"state": "MI",
				"zip": "48170"
			},
			"dob": "19470217",
			"first_name": "FTest2",
			"gender": "F",
			"insurance_plan": {
				"group_id": "10000026",
				"group_name": "NON-GROUP PROGRAM SENIOR PLUS",
				"plan_id": "MHMI0007",
				"subgroup_id": "1006",
				"subgroup_name": "NON-GROUP PROGRAM SENIOR PLUS MA"
			},
			"last_name": "LTest2",
			"medicare_number": "5P74WN8CG91",
			"member_suffix": "00",
			"middle_initial": "T",
			"phone_home": "2482771759",
			"provider": {
				"last_name": "VALENTI, MATTEO",
				"npi": "1912991894"
			},
			"relationship_code": "M",
			"subscriber_id": "100984456"
		}
	},
	{
		"patient": {
			"address": {
				"city": "ROMULUS",
				"country": "USA",
				"line1": "16239 TAFT ST",
				"state": "MI",
				"zip": "48174"
			},
			"dob": "19481203",
			"first_name": "FTest3",
			"gender": "F",
			"insurance_plan": {
				"group_id": "10000672",
				"group_name": "GENERAL MOTORS VEBA HMO",
				"plan_id": "MHMG0022",
				"subgroup_id": "1206",
				"subgroup_name": "GENERAL MOTORS VEBA PROTECTED SURV SP SEMI MA"
			},
			"last_name": "LTest3",
			"medicare_number": "7TJ3UC2EP87",
			"member_suffix": "00",
			"middle_initial": "L",
			"phone_home": "7347715452",
			"provider": {
				"last_name": "ATHENS, ELLEN K.",
				"npi": "1760444228"
			},
			"relationship_code": "M",
			"subscriber_id": "100116285"
		}
	},
	{
		"patient": {
			"address": {
				"city": "Ecorse",
				"country": "USA",
				"line1": "38 CHERRY GROVE",
				"state": "MI",
				"zip": "48229"
			},
			"dob": "19620411",
			"first_name": "FTest4",
			"gender": "F",
			"insurance_plan": {
				"group_id": "10000721",
				"group_name": "SUBURBAN MOBILITY AUTHORITY HMO",
				"plan_id": "HHMR0001",
				"subgroup_id": "1201",
				"subgroup_name": "SMART B-RETIREE"
			},
			"last_name": "LTest4",
			"member_suffix": "02",
			"provider": {
				"last_name": "ZAKOLSKI, DOUGLAS J.",
				"npi": "1659356400"
			},
			"relationship_code": "W",
			"subscriber_id": "100430131"
		}
	},
	{
		"patient": {
			"address": {
				"city": "LINCOLN PARK",
				"country": "USA",
				"line1": "2105 OLD GODDARD ROAD",
				"state": "MI",
				"zip": "48146"
			},
			"dob": "19610929",
			"first_name": "FTest5",
			"gender": "F",
			"insurance_plan": {
				"group_id": "10002562",
				"group_name": "DEARBORN HOLDING COMPANY LLC",
				"plan_id": "HHMR0001",
				"subgroup_id": "1100",
				"subgroup_name": "DEARBORN HOLDING COMPANY"
			},
			"last_name": "LTest5",
			"member_suffix": "01",
			"phone_home": "3133183138",
			"provider": {
				"last_name": "JURASEK, HENRY M.",
				"npi": "1568413367"
			},
			"relationship_code": "W",
			"subscriber_id": "101336074"
		}
	},
	{
		"patient": {
			"address": {
				"city": "LINCOLN PARK",
				"country": "USA",
				"line1": "1708 KINGS HIGHWAY",
				"state": "MI",
				"zip": "48146"
			},
			"dob": "19890516",
			"first_name": "FTest6",
			"gender": "F",
			"insurance_plan": {
				"group_id": "10000011",
				"group_name": "FORD MOTOR COMPANY HMO",
				"plan_id": "HEMR0001",
				"subgroup_id": "1009",
				"subgroup_name": "FORD MOTOR COMPANY - SALARIED HE COBRA"
			},
			"last_name": "LTest6",
			"member_suffix": "00",
			"provider": {
				"last_name": "BLANZY, JOHN E.",
				"npi": "1629017207"
			},
			"relationship_code": "M",
			"subscriber_id": "100675472"
		}
	},
	{
		"patient": {
			"address": {
				"city": "TEMPERANCE",
				"country": "USA",
				"line1": "1733 HEATHER DR",
				"state": "MI",
				"zip": "481829422"
			},
			"dob": "19400820",
			"first_name": "FTest7",
			"gender": "M",
			"insurance_plan": {
				"group_id": "10000026",
				"group_name": "NON-GROUP PROGRAM SENIOR PLUS",
				"plan_id": "MHPI0005",
				"subgroup_id": "1006",
				"subgroup_name": "NON-GROUP PROGRAM SENIOR PLUS MA"
			},
			"last_name": "LTest7",
			"medicare_number": "3KV9XE7NN45",
			"member_suffix": "00",
			"phone_home": "7342247818",
			"provider": {
				"last_name": "SEKHON, AVTAR S.",
				"npi": "1760536528"
			},
			"relationship_code": "M",
			"subscriber_id": "100135828"
		}
	},
	{
		"patient": {
			"address": {
				"city": "FARMINGTON HILLS",
				"country": "USA",
				"line1": "24823 COBBLESTONE CT",
				"state": "MI",
				"zip": "48336"
			},
			"dob": "20020803",
			"first_name": "FTest8",
			"gender": "M",
			"insurance_plan": {
				"group_id": "10000703",
				"group_name": "FCA US LLC HMO",
				"plan_id": "HHMR0001",
				"subgroup_id": "1000",
				"subgroup_name": "FCA US LLC ACTIVE"
			},
			"last_name": "LTest8",
			"member_suffix": "02",
			"middle_initial": "L",
			"phone_home": "3138150119",
			"provider": {
				"last_name": "CREGER, CLARK M.",
				"npi": "1699904086"
			},
			"relationship_code": "S",
			"subscriber_id": "100040659"
		}
	},
	{
		"patient": {
			"address": {
				"city": "REDFORD",
				"country": "USA",
				"line1": "9180 KINLOCH",
				"state": "MI",
				"zip": "48239"
			},
			"dob": "19970612",
			"first_name": "FTest9",
			"gender": "M",
			"insurance_plan": {
				"group_id": "10000374",
				"group_name": "CADON PLATING HMO",
				"plan_id": "HHMR0001",
				"subgroup_id": "1000",
				"subgroup_name": "CADON PLATING HMO ACTIVE"
			},
			"last_name": "LTest9",
			"member_suffix": "03",
			"middle_initial": "C",
			"phone_home": "7348377949",
			"provider": {
				"last_name": "VINTON, BRIAN A.",
				"npi": "1598053142"
			},
			"relationship_code": "S",
			"subscriber_id": "100263770"
		}
	},
	{
		"patient": {
			"address": {
				"city": "NORTHVILLE",
				"country": "USA",
				"line1": "17567 CRANBROOK DR",
				"state": "MI",
				"zip": "481684331"
			},
			"dob": "19931105",
			"first_name": "FTest10",
			"gender": "M",
			"insurance_plan": {
				"group_id": "10000005",
				"group_name": "HAP HMO INDIVIDUAL",
				"plan_id": "HYMR0310",
				"subgroup_id": "1000",
				"subgroup_name": "ON EXCHANGE"
			},
			"last_name": "LTest10",
			"member_suffix": "01",
			"phone_home": "2483440077",
			"provider": {
				"last_name": "KOTTUKAPALLY, NOBIN J.",
				"npi": "1942433289"
			},
			"relationship_code": "S",
			"subscriber_id": "100874961"
		}
	},
	{
		"patient": {
			"address": {
				"city": "BROWNSTOWN",
				"country": "USA",
				"line1": "24794 STREWING RD",
				"state": "MI",
				"zip": "48134"
			},
			"dob": "19900512",
			"first_name": "FTest11",
			"gender": "F",
			"insurance_plan": {
				"group_id": "10000017",
				"group_name": "FEDERAL EMPLOYEES HEALTH BENEFITS PROG.",
				"plan_id": "HHMR0001",
				"subgroup_id": "5600",
				"subgroup_name": "FEHB-97381600 DEPT VET AFFAIRS ACTIVE"
			},
			"last_name": "LTest11",
			"member_suffix": "00",
			"middle_initial": "J",
			"phone_home": "7347759091",
			"provider": {
				"last_name": "OOSTENDORP, MICHAELE C.",
				"npi": "1154329423"
			},
			"relationship_code": "M",
			"subscriber_id": "100008409"
		}
	},
	{
		"patient": {
			"address": {
				"city": "DETROIT",
				"country": "USA",
				"line1": "9583 GRANDMONT",
				"state": "MI",
				"zip": "48227"
			},
			"dob": "19390509",
			"first_name": "FTest12",
			"gender": "M",
			"insurance_plan": {
				"group_id": "10000026",
				"group_name": "NON-GROUP PROGRAM SENIOR PLUS",
				"plan_id": "MHMI0007",
				"subgroup_id": "1006",
				"subgroup_name": "NON-GROUP PROGRAM SENIOR PLUS MA"
			},
			"last_name": "LTest12",
			"medicare_number": "5XD5U79JV67",
			"member_suffix": "00",
			"provider": {
				"last_name": "ARUNACHALAM, VIJAYALAKSHMI",
				"npi": "1790764587"
			},
			"relationship_code": "M",
			"subscriber_id": "100821488"
		}
	},
	{
		"patient": {
			"address": {
				"city": "DEARBORN",
				"country": "USA",
				"line1": "3811 BENNET ST",
				"state": "MI",
				"zip": "48124"
			},
			"dob": "19590831",
			"first_name": "FTest13",
			"gender": "M",
			"insurance_plan": {
				"group_id": "10002581",
				"group_name": "ROBERT BOSCH CORPORATION HMO",
				"plan_id": "HHMR0001",
				"subgroup_id": "1000",
				"subgroup_name": "ROBERT BOSCH CORPORATION ACTIVE"
			},
			"last_name": "LTest13",
			"member_suffix": "00",
			"middle_initial": "M",
			"phone_home": "3132773162",
			"provider": {
				"last_name": "MEHDI, FARAH",
				"npi": "1033189477"
			},
			"relationship_code": "M",
			"subscriber_id": "100219147"
		}
	},
	{
		"patient": {
			"address": {
				"city": "MONROE",
				"country": "USA",
				"line1": "6816 SOUTH STONEY CREEK",
				"state": "MI",
				"zip": "48162"
			},
			"dob": "19550121",
			"first_name": "FTest14",
			"gender": "M",
			"insurance_plan": {
				"group_id": "10000703",
				"group_name": "FCA US LLC HMO",
				"plan_id": "HHMR0001",
				"subgroup_id": "1000",
				"subgroup_name": "FCA US LLC ACTIVE"
			},
			"last_name": "LTest14",
			"member_suffix": "00",
			"middle_initial": "R",
			"phone_home": "7348721190",
			"provider": {
				"last_name": "BELL, STEPHEN R.",
				"npi": "1184704181"
			},
			"relationship_code": "M",
			"subscriber_id": "100144046"
		}
	},
	{
		"patient": {
			"address": {
				"city": "TRENTON",
				"country": "USA",
				"line1": "3045 LAFAYETTE",
				"state": "MI",
				"zip": "48183"
			},
			"dob": "19560921",
			"first_name": "FTest15",
			"gender": "M",
			"insurance_plan": {
				"group_id": "10000673",
				"group_name": "CHRYSLER VEBA HMO",
				"plan_id": "MHMG0022",
				"subgroup_id": "1206",
				"subgroup_name": "CHRYSLER HOURLY VEBA GENERAL SEMI MA"
			},
			"last_name": "LTest15",
			"medicare_number": "8QE1EX8WV98",
			"member_suffix": "00",
			"middle_initial": "L",
			"phone_home": "7343082899",
			"provider": {
				"last_name": "JURASEK, HENRY M.",
				"npi": "1568413367"
			},
			"relationship_code": "M",
			"subscriber_id": "100106231"
		}
	},
	{
		"patient": {
			"address": {
				"city": "FLAT ROCK",
				"country": "USA",
				"line1": "26620 LARAMIE",
				"state": "MI",
				"zip": "48134"
			},
			"dob": "19561005",
			"first_name": "FTest16",
			"gender": "M",
			"insurance_plan": {
				"group_id": "10000586",
				"group_name": "METAL TECH PRODUCTS",
				"plan_id": "HHMR0007",
				"subgroup_id": "1000",
				"subgroup_name": "METAL TECH PRODUCTS ACTIVE"
			},
			"last_name": "LTest16",
			"member_suffix": "00",
			"middle_initial": "W",
			"phone_home": "3136082509",
			"provider": {
				"last_name": "NELSON, CAROLYN D.",
				"npi": "1164464301"
			},
			"relationship_code": "M",
			"subscriber_id": "100081887"
		}
	},
	{
		"patient": {
			"address": {
				"city": "TAYLOR",
				"country": "USA",
				"line1": "25616 KINYON ST",
				"state": "MI",
				"zip": "48180"
			},
			"dob": "19821014",
			"first_name": "FTest17",
			"gender": "M",
			"insurance_plan": {
				"group_id": "10002782",
				"group_name": "WYANDOTTE ALARM COMPANY",
				"plan_id": "HBMR0035",
				"subgroup_id": "1100",
				"subgroup_name": "WYANDOTTE ALARM COMPANY HMO 0 ACTIVE"
			},
			"last_name": "LTest17",
			"member_suffix": "00",
			"phone_home": "7349341630",
			"provider": {
				"last_name": "ROSSI, BENJAMIN M.",
				"npi": "1609818582"
			},
			"relationship_code": "M",
			"subscriber_id": "100300283"
		}
	},
	{
		"patient": {
			"address": {
				"country": "48239",
				"line2": "9336 COLUMBIA",
				"state": "REDFORD",
				"zip": "MI"
			},
			"first_name": "FTest18",
			"gender": "19730105",
			"insurance_plan": {
				"group_name": "10000664",
				"plan_id": "WAYNE STATE UNIVERSITY -PLAN 1 ACTIVE",
				"subgroup_id": "WAYNE STATE UNIVERSITY HMO",
				"subgroup_name": "1100"
			},
			"last_name": "LTest18",
			"medicare_number": "Y",
			"member_suffix": "00",
			"middle_initial": "VENUS",
			"phone_home": "USA",
			"phone_work": "3137217050",
			"primary_language": "F",
			"provider": {
				"last_name": "1821042581"
			},
			"relationship_code": "M",
			"subscriber_id": "100110932"
		}
	},
	{
		"patient": {
			"address": {
				"city": "CANTON",
				"country": "USA",
				"line1": "41486 SIMCOE DR",
				"state": "MI",
				"zip": "48188"
			},
			"dob": "19721124",
			"first_name": "FTest19",
			"gender": "M",
			"insurance_plan": {
				"group_id": "20000923",
				"group_name": "CALO AND SONS CONSTRUCTION INC",
				"plan_id": "HBMR0133",
				"subgroup_id": "1000",
				"subgroup_name": "CALO AND SONS CONSTRUCTION HMO 2500 PLUS ACTIVE"
			},
			"last_name": "LTest19",
			"member_suffix": "00",
			"middle_initial": "J",
			"provider": {
				"last_name": "SCHAEFFER, MICHAEL S.",
				"npi": "1780657569"
			},
			"relationship_code": "M",
			"subscriber_id": "100930402"
		}
	},
	{
		"patient": {
			"address": {
				"city": "MONROE",
				"country": "USA",
				"line1": "527 IDA MAYBEE RD",
				"state": "MI",
				"zip": "48161"
			},
			"dob": "20011024",
			"first_name": "FTest20",
			"gender": "F",
			"insurance_plan": {
				"group_id": "10000011",
				"group_name": "FORD MOTOR COMPANY HMO",
				"plan_id": "HHMR0001",
				"subgroup_id": "1000",
				"subgroup_name": "FORD MOTOR COMPANY - HOURLY ACTIVE"
			},
			"last_name": "LTest20",
			"member_suffix": "01",
			"middle_initial": "A",
			"phone_home": "3135873665",
			"provider": {
				"last_name": "ELLADKI, RACHA",
				"npi": "1043660640"
			},
			"relationship_code": "D",
			"subscriber_id": "100162768"
		}
	}
]
